src/i18n/locales/ca.json

```diff
@@ -90,12 +90,12 @@
     "process_count_other": "<0>Processos:</0> {{count}}",
     "view_profile": "Veure perfil"
   },
-  "organizations": {
-    "organizations_count_one": "Total {{ count }} organització",
-    "organizations_count_many": "",
-    "organizations_count_other": "Total {{ count }} organitzacions",
-    "organizations_list": "Organitzacions",
-    "search_by_org_id": "Cercar per identificador d'organització"
+  "accounts": {
+    "accounts_count_one": "Total {{ count }} compta",
```

Comment:
Compte, not compta

In these kind of cases btw, I personally write the entire translation (not passing {{ count }}):

`Total un únic compte`